### DISCUSSION TITLE
Web bluetooth pairing acting up (DC)

### DISCUSSION THREAD

**Discussion by Karabaskarbofos** [darthcloud/BlueRetro#340] [Q&A]
Hello dear Darthcloud. I'd like to bring up a weird issue. The story so far: I've done a DC internal mod according to this guide: 
![esp32-DREAMCAST](https://user-images.githubusercontent.com/110139085/181390768-78db948f-9eed-430f-ba62-66fdee746a30.png)
I pair ESP32 to the web server via Bluetooth, set some config and then nothing happens. Esp32  isn't recognised anymore when DC is booted up, only during the intro swirl. Cutting 5v and providing usb power helps restore the connection, but only with DC power off which interrupts the pairing with an error. Is there maybe something wrong with soldering or anything else? Note: controller port board is VA0 Japan, black wire is 5v and white is GND, temporarily desoldered to provide usb power.
![16589659627207010131981906043477](https://user-images.githubusercontent.com/110139085/181391766-8b149e8b-2ecf-44bf-a20d-73386e36131f.jpg)
![16589660225979067755850439849451](https://user-images.githubusercontent.com/110139085/181391857-fb6fde79-23d9-4a3d-9d97-3b6d78916975.jpg)

**Comment by darthcloud** [comment]
I got both VA0 and VA1 Dreamcast working fine personally.

Your soldering job look bad, too long expose wire and residue all over, do it again make sure to strip minimum length wire and cleanup up the board with alcohol after. 

try getting an official DevKit-C (Get it on [Digikey](https://www.digikey.com/en/products/detail/espressif-systems/ESP32-DEVKITC-32E/12091810?s=N4IgTCBcDaIKYGcAOBmMBaAJnAbgawEsAXAY3TThAF0BfIA) or [Mouser](https://www.mouser.com/ProductDetail/Espressif-Systems/ESP32-DevKitC-32E?qs=sGAEpiMZZMv0NwlthflBi3XYxq%252BTapwyHnR2%2FrAf4bY%3D)) 

Moving to discussion
